src/components/results/sensitivity-analysis/paged-sensitivity-analysis-result.js

```diff
@@ -39,6 +42,9 @@ const PagedSensitivityAnalysisResult = ({
     const [count, setCount] = useState(0);
     const [result, setResult] = useState(null);
     const [isLoading, setIsLoading] = useState(false);
+    const sensiStatus = useSelector(
+        (state) => state.computingStatus[ComputingType.SENSITIVITY_ANALYSIS]
```

Comment:
you can simply destruct 'computingStatus
``const sensiStatus = useSelector(
({computingStatus}) => computingStatus[ComputingType.SENSITIVITY_ANALYSIS]
);``

Reply:
> you can simply destruct 'computingStatus `const sensiStatus = useSelector( ({computingStatus}) => computingStatus[ComputingType.SENSITIVITY_ANALYSIS] );`

Personally I prefer the 1st way tin order o be consistent with the rest of the application.

Reply:
I prefer that it be consistent with what we currently have